addons/spreadsheet/static/src/pivot/plugins/pivot_core_plugin.js

```diff
@@ -1,29 +1,21 @@
 /** @odoo-module */
-
+//@ts-check
```

Comment:
Only this file is checked ? 😛 

Is the end goal to put `@ts-check` on all our (good) files, or to enable it in the js-config and hope every file donsn't turn red ?